README.md

```diff
@@ -98,5 +110,18 @@ effectiveOdds(
     { stake: 10, odds: { fractional: '2/1' }, terms: '1/4' }
   ]
 );
-// $> { decimal: 598.5, fractional: '1195/2' }
+// $> { decimal: 598.5, fractional: '597.5/1' }
 ```
+
+## Contributing
+
+### Build
+
+`> rollup -c rollup.config.js`
+
+### Minify
+
+`> npm install -g uglify-es`
+
+`> uglifyjs dist/bet-return.js -o dist/bet-return.min.js`
+
```

Comment:
I'm not sure these two steps are instrumental to contributing, as they would only need to be run when we're building a release (and would happen as part of the build script). The `test` and `test:cover` ones might be useful though.

Reply:
Good idea on the build script. I'm not going to create a build script here and now though, and I think it's worth leaving these instructions here until that build script is added.